src/quicklist.c

```diff
@@ -502,23 +510,51 @@ int quicklistNodeExceedsLimit(int fill, size_t new_sz, unsigned int new_count) {
     redis_unreachable();
 }
 
-/* Determines whether a given size qualifies as a large element based on a threshold
- * determined by the 'fill'. If the size is considered large, it will be stored in
+/* Determine whether the quicklist node has reached its limit before performing
+ * insertions, merges, or other operations that would increase the node size.
+ * Return 1 if exceeds the limit, otherwise 0. */
+REDIS_STATIC int quicklistNodeExceedsLimit(const quicklist *ql, size_t new_sz,
+                                           unsigned int new_count) {
```

Comment:
```suggestion
REDIS_STATIC int quicklistNodeExceedsLimit(const quicklist *ql, size_t new_sz, unsigned int new_count) {
```
it's acceptable to make 104 columns in one row.